src/Commands/Run.php

```diff
@@ -21,9 +22,7 @@ class Run extends Command
      * @var string
      */
     protected $signature = "tenants:run {commandname : The command's name.}
```

Comment:
Perhaps change this to:
```suggestion
    protected $signature = "tenants:run {command : The artisan command.}
```
So that it's clear that it's not just the *name* but also the options and arguments.

Reply:
Thanks for the review. I've updated the `commandname`'s description. But looks like we can't use the `command` as a name because it's being used internally.
<img width="430" alt="image" src="https://user-images.githubusercontent.com/54532330/182778551-71634e70-df24-4cb5-baf9-ddb26ac98c17.png">

Reply:
I see, that's fine. Thanks for checking that, I'll merge this with `commandname` then